examples/st7789/main.go

```diff
@@ -9,12 +9,37 @@ import (
 )
 
 func main() {
+
+	// Example configuration for Adafruit Clue
+	// machine.SPI1.Configure(machine.SPIConfig{
+	//	Frequency: 8000000,
+	//	SCK:       machine.TFT_SCK,
+	//	SDO:       machine.TFT_SDO,
+	//	SDI:       machine.TFT_SDO,
+	//	Mode:      0,
+	// })
+	// display := st7789.New(machine.SPI1,
+	//	machine.TFT_RESET,
+	//	machine.TFT_DC,
+	//	machine.TFT_CS,
+	//	machine.TFT_LITE)
```

Comment:
Is there a best practice for handling hardware/machine specific recommendations in the device examples?

Reply:
I'm not sure if it's best practice, but I would do the following

https://github.com/tinygo-org/drivers/tree/release/examples/ili9341/pyportal_boing
https://github.com/tinygo-org/tinyfont/tree/dev/examples/initdisplay